root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,214 @@
+            {
+                sessionAbandonFlag = (bool)Session[FxHttpContextIndex.SESSION_ABANDON_FLAG];
+            }
+
+            Session.Remove(FxHttpContextIndex.SESSION_ABANDON_FLAG);
+
+            // ------------------------------------------------------------------------
+
+            //画面にフォーム情報を表示する---------------------------------------------
+            NameValueCollection form = (NameValueCollection)Session[FxHttpContextIndex.FORMS_INFORMATION];
+            Session.Remove(FxHttpContextIndex.FORMS_INFORMATION);
+
+            if (form != null)
+            {
+                //foreach
+                foreach (string strKey in form)
+                {
+                    if (form[strKey] == null)
+                    {
+                        //Add key and value to PositionData
+                        list_form.Add(new PositionData(strKey, "null"));
+                    }
+                    else
+                    {
+                        //Add key and value to PositionData
+                        list_form.Add(new PositionData(strKey, CustomEncode.HtmlEncode(form[strKey].ToString())));
+                    }
+                }
+                //データバインド
+                ViewBag.list_form = list_form;
+            }
+
+            // 画面にセッション情報を表示する------------------------------------------
+
+            if (Session != null)
+            {
+                //foreach
+                foreach (string strKey in Session)
+                {
+                    if (Session[strKey] == null)
+                    {
+                        //Add key and value to PositionData
+                        list_session.Add(new PositionData(strKey, "null"));
+                    }
+                    else
+                    {
+                        //Add key and value to PositionData
+                        list_session.Add(new PositionData(strKey, CustomEncode.HtmlEncode(Session[strKey].ToString())));
+                    }
+                }
+                //データバインド
+                ViewBag.list_session = list_session;
+            }
+
+            // セッション情報を削除する------------------------------------------------
+
+            if (sessionAbandonFlag)
+            {
+                // セッション タイムアウト検出用Cookieを消去
+                // ※ Removeが正常に動作しないため、値を空文字に設定 ＝ 消去とする
+
+                // Set-Cookie HTTPヘッダをレスポンス
+                Response.Cookies.Set(FxCmnFunction.DeleteCookieForSessionTimeoutDetection());
+
+                try
+                {
+                    // セッションを消去                       
+                    Session.Abandon();
+                }
+                catch (Exception ex)
+                {
+                    // エラー発生時
+                    // このカバレージを通過する場合、
+                    // おそらく起動した画面のパスが間違っている。
+                    Console.WriteLine("このカバレージを通過する場合、おそらく起動した画面のパスが間違っている。");
+                    Console.WriteLine(ex.Message);
```

Comment:
@daisukenishino 

Console window does not exist with ASP.NET web application.
So, the messages that is outputted by `Console.WriteLine` cannot be notified to the users.